docs/zh-CN/sql-reference/sql-statements/Data Manipulation/ROUTINE LOAD.md

```diff
@@ -310,6 +318,84 @@ under the License.
             "property.client.id" = "my_client_id"
         );
 
+    4. 为 example_db 的 example_tbl 创建一个名为 test1 的 Kafka 例行导入任务，导入的简单json数据。
+        1）数据样例, doris_data为固定关键字
+        {
+            "doris_data":[
+                {"category":"a9jadhx","author":"test","price":895},
+                {"category":"axdfa1","author":"EvelynWaugh","price":1299}
+            ]
+        }
+        2) 创建任务，可不设置jsonpath或者jsonpath_file
+        CREATE ROUTINE LOAD example_db.test1 ON example_tbl
+        COLUMNS(category, author, price)
+        PROPERTIES
+        (
+            "desired_concurrent_number"="3",
+            "max_batch_interval" = "20",
+            "max_batch_rows" = "300000",
+            "max_batch_size" = "209715200",
+            "strict_mode" = "false",
+            "format" = "json"
+        )
+        FROM KAFKA
+        (
+            "kafka_broker_list" = "broker1:9092,broker2:9092,broker3:9092",
+            "kafka_topic" = "my_topic",
+            "kafka_partitions" = "0,1,2",
+            "kafka_offsets" = "0,0,0"
+        );
+
+    5. 通过jsonpath参数，为 example_db 的 example_tbl 创建一个名为 test1 的 Kafka 例行导入任务，导入的数据格式为json。
+
+        CREATE ROUTINE LOAD example_db.test1 ON example_tbl
+        COLUMNS(category, author, price)
+        PROPERTIES
+        (
+            "desired_concurrent_number"="3",
+            "max_batch_interval" = "20",
+            "max_batch_rows" = "300000",
+            "max_batch_size" = "209715200",
+            "strict_mode" = "false",
+            "format" = "json",
+            "jsonpath" = "{\"jsonpath\":[{\"column\":\"category\",\"value\":\"$.store.book.category\"},{\"column\":\"author\",\"value\":\"$.store.book.author\"},,{\"column\":\"price\",\"value\":\"$.store.book.price\"}]}"
```

Comment:
Why not use `jsonpaths=["$.store.book.category", "$.store.book.author", "$.store.book.price"]`